src/components/app/TabBar.css

```diff
@@ -1,6 +1,6 @@
 /* Removing styling from the button */
 .tabBarTabButton {
-  padding: 0;
+  padding: 2px;
```

Comment:
I understand what you're trying to do here, but I think we should do it differently, for the following reasons:
1. this increases the height of the tabs
2. this is still not very nice, it would be better if the focus indicator was around the whole tab.

So there are a bunch of solutions I have:

* Probably the easiest: make the `<button>` take the whole space. I think you can do it if you remove all paddings in the tabbar, replace it with a fixed `height` somewhere, and then have the button take `width/height: 100%`
* otherwise remove the focus indicator off the `<button>` and use `:focus-within` on the tab itself.

But I think the first idea is better.

Reply:
To prevent the focus indicator from appearing in the tab bar when clicking, here is what I did and seems to work:
in TabBar.js add a `onMouseDown` handler that calls `e.preventDefault`. This was already in `onClickListener` but I think it wasn't working, so we can just move it:

```js
  _onClickListener = (e: SyntheticMouseEvent<HTMLElement>) => {
    this.props.onSelectTab(e.currentTarget.dataset.name);
  };

  _onMouseDown = (e: SyntheticMouseEvent<>) => {
    // Prevent focusing the tab so that actual content like the
    // calltree can perform its own focusing.
    e.preventDefault();
  };

  render() {
    ...
            <li
              ...
              onMouseDown={this._onMouseDown}
            >
  }

```